<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>691</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>빛의 계승자 Regular</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(42, 0, 149);color: white;</string>
    </property>
    <property name="text">
     <string>보기만 해도 지려버리는 한줄 번역기 (powered by Google Translate)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(80, 156, 255);</string>
    </property>
    <property name="text">
     <string>한국어 입력:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_ko">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid; border-color: rgb(64, 67, 122);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_trans">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>110</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(182, 229, 26);</string>
    </property>
    <property name="text">
     <string>번 역</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(80, 156, 255);</string>
    </property>
    <property name="text">
     <string>영어:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_en">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>741</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid; border-color: rgb(64, 67, 122);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(80, 156, 255);</string>
    </property>
    <property name="text">
     <string>일본어:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_ja">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>741</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid; border-color: rgb(64, 67, 122);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(80, 156, 255);</string>
    </property>
    <property name="text">
     <string>중국어:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_zh_cn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>741</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid; border-color: rgb(64, 67, 122);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>540</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(182, 229, 26);</string>
    </property>
    <property name="text">
     <string>초기화, ㅋ</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
